ztp/gitops-subscriptions/argocd/deployment/kustomization.yaml

```diff
@@ -8,3 +8,4 @@ resources:
   - gitops-cluster-rolebinding.yaml
   - clusters-app.yaml
   - policies-app.yaml
+  - talm-operator-subscription.yaml
```

Comment:
Based on feedback from QE, because the environment is likely disconnected and the catalog source name will vary) it may be best to not automatically include the Subscription in the deployment (so remove this line from the kustomization.yaml). Instead, can you work with the documentation team to update [this section](https://docs.openshift.com/container-platform/4.10/scalability_and_performance/ztp-deploying-disconnected.html#ztp-preparing-the-hub-cluster-for-ztp_ztp-deploying-disconnected) of the docs to refer the user to this sample Subscription as a way to install the TALM operator (pointing out the need to update the catalogsource as needed)